src/lambda/utilities/lambdaPayloadUtils.ts

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
This file is very special-case, can it live in the existing `src/lambda/utils.ts` ?